wremnants/muon_efficiencies_smooth.py

```diff
@@ -106,48 +107,62 @@ def make_muon_efficiency_helpers_smooth(filename = data_dir + "/testMuonSF/allSm
     # in this way the name of the (single) effType axis can be used to activate some internal behaviour as needed (for example trigger won't have variations for not triggering lepton)
     
     effStat_manager = {"sf_reco": {"nPtEigenBins" : 0,
+                                   "nCharges" : 0,
                                    "axisLabels" : ["reco"],
                                    "boostHist" : None,
                                    "helper" : None},
                        "sf_tracking": {"nPtEigenBins" : 0,
+                                       "nCharges" : 0,
                                        "axisLabels" : ["tracking"],
                                        "boostHist" : None,
                                        "helper" : None},
                        "sf_idip": {"nPtEigenBins" : 0,
+                                   "nCharges" : 0,
                                    "axisLabels" : ["idip"],
                                    "boostHist" : None,
                                    "helper" : None},
                        "sf_trigger": {"nPtEigenBins" : 0,
-                                   "axisLabels" : ["trigger"],
-                                   "boostHist" : None,
-                                   "helper" : None},
+                                      "nCharges" : 0,
```

Comment:
Why are nCharges (and nPt) set to 0 here?

Reply:
Ok nevermind, I see that they get updated later.  It might be better to put "None" instead of zero in that case, to guarantee that it errors out in case the update is somehow missed.